packages/cactus-plugin-ledger-connector-quorum/src/test/typescript/integration/plugin-ledger-connector-quorum/deploy-contract/v21.4.1-invoke-contract.test.ts

```diff
@@ -24,182 +24,196 @@ import { PluginRegistry } from "@hyperledger/cactus-core";
+    const testEthAccount = web3.eth.accounts.create(uuidV4());
 
-    const { callOutput: helloMsg } = await connector.getContractInfoKeychain({
-      contractName,
-      keychainId: keychainPlugin.getKeychainId(),
-      invocationType: EthContractInvocationType.Call,
-      methodName: "sayHello",
-      params: [],
-      web3SigningCredential: {
-        ethAccount: firstHighNetWorthAccount,
-        secret: "",
-        type: Web3SigningCredentialType.GethKeychainPassword,
-      },
+    const keychainEntryKey = uuidV4();
+    const keychainEntryValue = testEthAccount.privateKey;
+    const keychainPlugin = new PluginKeychainMemory({
+      instanceId: uuidV4(),
+      keychainId: uuidV4(),
+      // pre-provision keychain with mock backend holding the private key of the
+      // test account that we'll reference while sending requests with the
+      // signing credential pointing to this keychain entry.
+      backend: new Map([[keychainEntryKey, keychainEntryValue]]),
+      logLevel,
     });
-    t2.ok(helloMsg, "sayHello() output is truthy");
-    t2.true(
-      typeof helloMsg === "string",
-      "sayHello() output is type of string",
+    keychainPlugin.set(
+      HelloWorldContractJson.contractName,
+      JSON.stringify(HelloWorldContractJson),
+    );
+    // Instantiate connector with the keychain plugin that already has the
+    // private key we want to use for one of our tests
+    const connector: PluginLedgerConnectorQuorum = new PluginLedgerConnectorQuorum(
+      {
+        instanceId: uuidV4(),
+        rpcApiHttpHost,
+        logLevel,
+        pluginRegistry: new PluginRegistry({ plugins: [keychainPlugin] }),
+      },
     );
-  });
 
-  test("invoke Web3SigningCredentialType.GETHKEYCHAINPASSWORD", async (t2: Test) => {
-    const newName = `DrCactus${uuidV4()}`;
-    const setNameOut = await connector.getContractInfoKeychain({
-      contractName,
-      keychainId: keychainPlugin.getKeychainId(),
-      invocationType: EthContractInvocationType.Send,
-      methodName: "setName",
-      params: [newName],
+    await connector.transact({
       web3SigningCredential: {
         ethAccount: firstHighNetWorthAccount,
         secret: "",
         type: Web3SigningCredentialType.GethKeychainPassword,
       },
-      nonce: 2,
+      transactionConfig: {
+        from: firstHighNetWorthAccount,
+        to: testEthAccount.address,
+        value: 10e9,
+      },
     });
-    t2.ok(setNameOut, "setName() invocation #1 output is truthy OK");
 
-    try {
-      const setNameOutInvalid = await connector.getContractInfoKeychain({
+    const balance = await web3.eth.getBalance(testEthAccount.address);
+    expect(balance).toBeTruthy();
+    expect(parseInt(balance, 10)).toEqual(10e9);
+    // t.equals(parseInt(balance, 10), 10e9, "Balance of test account is OK");
+
+    let contractAddress: string;
+
+    {
+      // test("deploys contract via .json file", async (t2: Test) => {
```

Comment:
@Leeyoungone No commented code unless you need it for something specific please :-)